libres/lib/enkf/forward_load_context.cpp

```diff
@@ -95,11 +99,33 @@ forward_load_context_load_ecl_sum(forward_load_context_type *load_context) {
 
         if ((header_file != NULL) && (stringlist_get_size(data_files) > 0)) {
             bool include_restart = false;
-            bool lazy_load = true;
-            int file_options = 0;
-            summary = ecl_sum_fread_alloc(
-                header_file, data_files, SUMMARY_KEY_JOIN_STRING,
-                include_restart, lazy_load, file_options);
+
+            /*
+             * Setting this flag causes summary-data to be loaded by
+             * ecl::unsmry_loader which is "horribly slow" according
+             * to comments in the code. The motivation for introducing
+             * this mode was at some point to use less memory, but
+             * computers nowadays should not have a problem with that.
```

Comment:
This is probably fine, but have you tested this on some large cases to verify?

Reply:
Using the `EclSum::create_writer()` method it is quite simple create summary testcases for disk. I don't know whether a sufficiently large case will be sufficiently fast to create - but maybe worth a try?

Reply:
The effect of toggling this flag is brutal on any test larger than minimal ones. It was discovered during [this work](https://github.com/equinor/ert/pull/2704/files) by @mortalisk when developing a framework for creating tailored syntetic tests.